src/tests/audio/tasks/forced_alignment_test.py

```diff
@@ -182,7 +172,8 @@ def test_align_transcriptions_fixture(resampled_mono_audio_sample: Audio, script
     aligned_transcriptions = align_transcriptions(audios_and_transcriptions_and_language)
     assert len(aligned_transcriptions) == 2
     assert len(aligned_transcriptions[0]) == 1
-    assert aligned_transcriptions[0][0].text == "test"
+    if aligned_transcriptions[0][0]:
+        assert aligned_transcriptions[0][0].text == "test"
```

Comment:
@ibevers, how is the expected transcript text "test"?

Reply:
@fabiocat93 Good point--will change to match the audio file